forge-ai/src/main/java/forge/ai/ability/PeekAndRevealAi.java

```diff
@@ -73,6 +75,15 @@ protected boolean canPlayAI(Player aiPlayer, SpellAbility sa) {
      */
     @Override
     public boolean confirmAction(Player player, SpellAbility sa, PlayerActionConfirmMode mode, String message, Map<String, Object> params) {
+        if ("InstantOrSorcery".equals(sa.getParam("AILogic"))) {
+            Set<Card> revealed = AiCardMemory.getMemorySet(player, AiCardMemory.MemorySet.REVEALED_CARDS);
```

Comment:
Hmn, it would certainly be nice for the AI to change it's playstyle if it gains information about opponents hidden zones

But for this case it seems a bit roundabout to me as just modifying `PeekAndRevealEffect` to pass those cards via the params like in other cases would be easier? 🤷‍♂️

Reply:
Yeah, it just seemed like a good opportunity to implement the stubbed out reveal for possible future cases :) I might switch the Delver implementation into a params based one but keep the reveal stuff for other uses later. What do you think?

Reply:
Sure, maybe I'll see if we can get some quick wins with it in places like `hasAFogEffect` next

There is also `Game.findByView`

Reply:
Hmm, I tried a few things but I can't quite figure out how we can utilize a SA param for this... I mean, the SA param map maps from a String to a String, while the contents of the things to peek/reveal will either be a single Card object or even a collection of Card objects, if multiple are revealed at the same time. How can this be stored in the PeekAndRevealEffect in order to be grabbed later from confirmAction in the AI code?.. I'll probably need a few pointers here, sadly...

Reply:
Ah, that makes sense! :) Didn't see that one somehow, haha. Thanks!! :)

Reply:
Done, I think :)